app/views/sign_requests/index.html.erb

```diff
@@ -0,0 +1,30 @@
+<% provide(:title, "Request signs") %>
+
+<div class="grid-x list">
+  <div class="list__section cell grid-x">
+    <div class="cell small-12 large-9 margin-bottom-2">
+      <h1>Request signs</h1>
+    </div>
+
+    <div class="cell small-12 large-6">
+      <h3 class="margin-bottom-1">Request signs on Facebook</h3>
+
+      <p>
+        Intro and overview information on Facebook requests, and possible future integration into this website.
```

Comment:
Can we request content for this page from Micky, so that we can deploy this with static text while we wait for the next NZSL video recording session?

Reply:
It looks like some copy was handed to us yesterday. Will look at it now